### PR TITLE
Update ClayNavbar markup

### LINKED ISSUE #130: ClayNavbar
We need to update the markup

### PR DESCRIPTION
**Pull request by matuzalemsteles**
This resolves #130.

#### Changes in pr
* Add areaExpanded to ClayLink dbc79fb 332eab8
* Update the ClayNavbar Markup 189443e

### CODE REVIEW

**Comment by jbalsas** [comment]
Hey @carloslancha, I'm going to go ahead and merge this so we can keep going, but please, do review it and comment on that so @matuzalemsteles can send a followup PR if needed!

File: packages/clay-link/src/ClayLink.js [hunk 1]
```diff
@@ -26,6 +26,15 @@ ClayLink.STATE = {
...
 	ariaLabel: Config.string(),
 
+	/**
+	 * Aria Expanded attribute for the anchor element.
+	 * @instance
+	 * @memberof ClayLink
+	 * @type {?bool|undefined}
+	 * @default undefined
+	 */
+	areaExpanded: Config.bool(),
```
Comment (carloslancha) [inline packages/clay-link/src/ClayLink.js:36]: Is `areaExpanded` a typo error? Should it be `ariaExpanded`?

**Comment by carloslancha** [comment]
I'd rename this component to ClayNavigationBar to maintain consistency with claycss.com and lexicon

File: packages/clay-link/src/ClayLink.js (hunk 1 above)
Comment (matuzalemsteles) [reply packages/clay-link/src/ClayLink.js:36]: Yeah, went unnoticed 😅. Thanks!!

**Pull request merged**